resources/public/api/conf/1.0/examples/retrieveReliefInvestments/retrieve_relief_investments_response.json

```diff
@@ -0,0 +1,65 @@
+{
+  "vctSubscription":[
+    {
+      "uniqueInvestmentRef": "VCTREF",
+      "name": "VCT Fund X",
+      "dateOfInvestment": "2018-04-16",
+      "amountInvested": 23312.00,
+      "reliefClaimed": 1334.00
+    }
+  ],
+  "eisSubscription":[
+    {
+      "uniqueInvestmentRef": "XTAL",
+      "name": "EIS Fund X",
+      "knowledgeIntensive": true,
+      "dateOfInvestment": "2020-12-12",
+      "amountInvested": 23312.00,
+      "reliefClaimed": 43432.00
+    }
+  ],
+  "communityInvestment": [
+    {
+      "uniqueInvestmentRef": "CIREF",
+      "name": "CI X",
+      "dateOfInvestment": "2020-12-12",
+      "amountInvested": 6442.00,
+      "reliefClaimed": 2344.00
+    }
+  ],
+  "seedEnterpriseInvestment": [
+    {
+      "uniqueInvestmentRef": "123412/1A",
+      "companyName": "Company Inc",
+      "dateOfInvestment": "2020-12-12",
+      "amountInvested": 123123.22,
+      "reliefClaimed": 3432.00
+    }
+  ],
+  "socialEnterpriseInvestment": [
+    {
+      "uniqueInvestmentRef": "123412/1A",
+      "socialEnterpriseName": "SE Inc",
+      "dateOfInvestment": "2020-12-12",
+      "amountInvested": 123123.22,
+      "reliefClaimed": 3432.00
+    }
+  ],
+  "links":[
+    {
+      "href":"/reliefs/investment/{nino}/{taxYear}",
```

Comment:
replace `{nino}` and `{taxYear}` with example nino and taxYear (also change this in the other links)